backends-velox/src/test/scala/io/glutenproject/execution/VeloxAggregateFunctionsSuite.scala

```diff
@@ -371,6 +371,26 @@ abstract class VeloxAggregateFunctionsSuite extends VeloxWholeStageTransformerSu
     }
   }
 
+  test("regr_r2") {
+    assume(isSparkVersionAtleast("3.3"))
```

Comment:
Seems we can just use `testWithSpecifiedSparkVersion` to limit minimum Spark version?

Reply:
`testWithSpecifiedSparkVersion` cannot express >= some version. Also, will there be any changes for Spark 3.3 and Spark 3.4 in the future? For example, upgrading from Spark 3.3.1 to 3.3.4.

Reply:
@liujiayi771, cannot the below code work? It requires the minimum version=3.3, regardless of the third version digit. E.g., Spark's "3.3.1", "3.3.4" or "3.4.2" will pass the internal check (>= "3.3").
`testWithSpecifiedSparkVersion("name", "3.3")`

Reply:
@PHILO-HE Yes, this can work, thanks.